addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/command/UdpCubeCommand.java

```diff
@@ -112,8 +112,10 @@ private void receiveUdpCommandResponse() {
                 // We have a response
                 String message = new String(receivePacket.getData(), receivePacket.getOffset(),
                         receivePacket.getLength(), StandardCharsets.UTF_8);
-                logger.trace("Broadcast response from {} : {} '{}'", receivePacket.getAddress(), message.length(),
-                        message);
+                if (logger.isDebugEnabled()) {
+                    logger.debug("Broadcast response from {} : {} '{}'", receivePacket.getAddress(), message.length(),
```

Comment:
Is there a particular reason to make this debug instead of trace... 
Would not expect this is of much use for the average user or debugging.

Reply:
Well, I did expect it to be a debugging message, because it outputs a general data info, not a trace info (it doesn't say which part of the code is executed). So I thought I'd change it accordingly. However, if you insist, I can change it back.

Reply:
I can't be bothered too much for this one.

Reply:
Understood, but does that mean that you want to leave it or change it back?

Reply:
Sorry, I just saw your approval. Thanks.